tests/gen/cli/gendocs.go

```diff
@@ -123,7 +123,7 @@ func saveBatchToCollections(
 ) error {
 	for colName, colDocs := range colDocsMap {
 		for _, col := range collections {
-			if col.Description().Name == colName {
+			if col.Description().Name.Value() == colName {
```

Comment:
> question: Why have you skipped `HasValue()` here before calling `Value()`. If IRRC the value returned is invalid unless `HasValue() == true`

question: Same as above

Reply:
We only generate docs for named collections